app/src/main/res/layout/activity_main.xml

```diff
@@ -2,17 +2,24 @@
 <androidx.constraintlayout.widget.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
     xmlns:app="http://schemas.android.com/apk/res-auto"
     xmlns:tools="http://schemas.android.com/tools"
-    android:layout_width="match_parent"
-    android:layout_height="match_parent"
+    android:layout_width="wrap_content"
+    android:layout_height="wrap_content"
     tools:context=".MainActivity">
 
     <Button
         android:id="@+id/btnLaunchManager"
         android:layout_width="wrap_content"
         android:layout_height="wrap_content"
-        app:layout_constraintLeft_toLeftOf="parent"
-        app:layout_constraintTop_toTopOf="parent"
         android:text="@string/launch_dictionary_manager"
-        />
+        app:layout_constraintStart_toStartOf="parent"
+        app:layout_constraintTop_toTopOf="parent" />
+
+    <Button
+        android:id="@+id/btnLaunchDictionary"
+        android:layout_width="match_parent"
+        android:layout_height="match_parent"
```

Comment:
Change the height here to `wrap_content`. Haha, setting this to `match_parent` as you did, you were making the button's height *explode* to cover the entire screen!!!